tests/CMakeLists.txt

```diff
@@ -25,7 +25,9 @@ if(BUILD_GENIVI)
 endif(BUILD_GENIVI)
 
 add_custom_target(build_tests DEPENDS aktualizr)
-add_custom_target(check COMMAND CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=1 ${CMAKE_CTEST_COMMAND} DEPENDS build_tests)
+add_custom_target(check COMMAND CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=1 ${CMAKE_CTEST_COMMAND} -E test_valgrind_uptane_vectors\\|test_build DEPENDS build_tests)
+add_custom_target(check-full COMMAND CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=1 ${CMAKE_CTEST_COMMAND} -E test_uptane_vectors DEPENDS build_tests)
```

Comment:
Won't CTEST_PARALLEL_LEVEL=1 override the CTEST_PARALLEL_LEVEL=4 from tests.sh and coverage.sh?